### PR TITLE
Remove OrderedDict from dcim.models.racks.py

### PR DESCRIPTION
**Pull request by edwardtheharris**
# See #1746 

# What's Changed

The OrderedDict instance on line #275 of the file `nautobot/dcim/models/racks.py` has been replaced with a plain dictionary defined using curly braces.

# Screenshots

n/a

# TODO

- [x] Explanation of Change(s)
- [x] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [x] Attached Screenshots, Payload Example
- [x] Unit, Integration Tests
- [x] Documentation Updates (when adding/changing features)
- [x] Example Plugin Updates (when adding/changing features)
- [x] Outline Remaining Work, Constraints from Design

### CODE REVIEW

File: changes/1746.housekeeping [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Replaced `OrderedDict` instance in `nautobot/models/racks.py#275` with a plain
+`dict` instance.
```
Comment (glennmatthews) [inline changes/1746.housekeeping:2]: Line breaks in change fragments translate to separate entries in the release-notes, so this needs to be on one line:

```suggestion
Replaced `OrderedDict` instance in `nautobot/models/racks.py#275` with a plain `dict` instance.
```

**Review by glennmatthews — APPROVED** [review]
Thanks for the cleanup!

**Comment by edwardtheharris** [comment]
> Thanks for the cleanup!

No worries, happy to help.

File: changes/1746.housekeeping (hunk 1 above)
Comment (glennmatthews) [reply changes/1746.housekeeping:2]: Whoops, looks like this got reverted!

Comment (edwardtheharris) [reply changes/1746.housekeeping:2]: I think I inadvertently pulled in changes from the wrong develop branch and it mucked things up a bit, apologies for that. Should be good to go now though.

**Pull request merged**